### ISSUE TITLE
[ENHANCEMENT] "Continue Thread" to load comments inline (like v19) rather than load new page when comment indentation set to "Classic"

### ISSUE THREAD

**Issue by LegaliseAllDrugs** [laurencedawson/sync-for-reddit#1583]
**Is your feature request related to a problem? Please describe.**  
In v22, the "classic" style of comment indentation doesn't include the classic functionality.

In v22, if a user has the comment indentation setting set to "sync classic", and then taps "continue thread" to load more comments in a thread, this loads a new page with only the next handful of comments, rather than loading them in-line like v19 used to do.

**Describe the solution you'd like**  
If a user has comment indentation set to "sync classic", I would love for comments to load in-line like they used to in v19.

**Additional context**  
How it used to work on v19:
![Screenshot_20220808-145410_Sync](https://user-images.githubusercontent.com/110836374/183445576-ad71e7e2-e1a2-40e8-86a8-db720cb1ce94.jpg)

The indentation was so slim that a user could view a thread 70 comments deep and they would still be legible. 
![Screenshot_20220808-143621_Sync](https://user-images.githubusercontent.com/110836374/183446431-1952b7d7-f02c-4d2e-b6a1-3ee657ae8efc.jpg)

This was, to me, the single greatest strength of Sync over every other app for Reddit. It was even superior to the desktop web experience.   It was far more pleasant an experience than clunkily navigating between various pages. Please let it be an option in v22.

**Comment by laurencedawson** [comment]
The main issue with this was that the thread would get super squished over time...

**Comment by LegaliseAllDrugs** [comment]
Yes but, as shown in second screenshot, even after something like 80 or 90 comments it's still very readable. Beyond that I would tap on a comment and load it on a new page (either by "opening in browser" or tapping share>sync>link handler) to solve that problem. Any day of the week I would opt for this experience over a hard limit of 10 comments per page.

**Comment by LegaliseAllDrugs** [comment]
@laurencedawson Please, I beg you. Can you please provide the option to load new comments inline like v19? At least the first 60 or 70 comments, before it navigates to a new page? It is to me the single greatest strength of Sync over any other reddit experience, including the desktop website.

**Comment by FaviFake** [comment]
> I would tap on a comment and load it on a new page (either by "opening in browser" or tapping share>sync>link handler) to solve that problem.

@LegaliseAllDrugs This is way too complicated, normal users are never going to do that. The text is too squished and hard to read, and there isn't a fast way and easy way to make it easier to read

**Comment by LegaliseAllDrugs** [comment]
@FaviFake Many, many normal users currently use Sync v19 and are very happy with it. So it isn't as though the way it handles comments is an overwhelming problem. All I want is for the option to use the v19 system in v22, because I think it is superior. 

If you want to make it even better with a fast and easy way to unsquish it, then just have an option to have it open a new page after X comments instead of after just 10. Or have an additional button alongside "continue thread" that says "continue thread on new page" or something similar. Or set a minimum comment width and have all the comments automatically shift leftward as you scroll through a long thread. Many possible solution that are better than a hard limit of 10 comments per page.
